api/src/main/java/org/openmrs/module/patientflags/PatientFlagsModuleActivator.java

```diff
@@ -85,7 +87,9 @@ public void started() {
 		}
 		
 		// set the new names
-		thisModule.setExtensionNames(extensionPoints);

```

Comment:
The method was not removed. The parameter was rather changed from `IdentityHashMap` to `Map`.
So, you need to deal with those two cases.

Reply:
@dkayiwa any examples available

Reply:
Is this of help? https://github.com/openmrs/openmrs-module-xforms/commit/5d79f327d27777a40ec755a90433e4dc69067e4a

Reply:
Why don't you make it as simple as this? https://pastebin.com/0TMgGhKg

Reply:
@dkayiwa  thanks. I believe its done

Reply:
Does the solution make sense to you?

Reply:
yes it does. It was close.